### ISSUE TITLE
Display tab in Mate window containing only one terminal

### ISSUE THREAD

**Issue by bergstenarn** [mate-desktop/mate-terminal#420]
#### Expected behaviour
Tabs shall always be displayed, even if there's only one terminal in the Mate window.

#### Actual behaviour
When the Mate window contains only one terminal it's not possible to drag that terminal to another Mate window.

#### Steps to reproduce the behaviour
Start two Mate windows. Try to drag the terminal in the first window over to the second. It doesn't work ... or I haven't been able to do that despite plenty of attempts.

#### MATE general version
$ mate-terminal --version
MATE Terminal 1.18.1

#### Package version
$ zypper search -is mate-terminal
Loading repository data...
Reading installed packages...

S  | Name          | Type    | Version    | Arch   | Repository                  
---+---------------+---------+------------+--------+-----------------------------
i+ | mate-terminal | package | 1.18.1-1.1 | x86_64 | The MATE desktop environment

#### Linux Distribution
$ cat /etc/os-release 
NAME="SLES"
VERSION="12-SP2"
VERSION_ID="12.2"
PRETTY_NAME="SUSE Linux Enterprise Server 12 SP2"
ID="sles"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:suse:sles:12:sp2"

#### Link to bugreport of your Distribution (requirement)
Don't know where to find that bugreport or how to generate it. Sorry. Hope that's ok as this is more kind of a feature request than a bug.

**Comment by lukefromdc** [comment]
Users who do not use tabs in terminal (myself included) may not want to show the tab bar unconditionally.

**Comment by cwendling** [comment]
Yeah, it at least should be an option, I surely wouldn't like the space wasted in my 1-tab windows.

However, isn't the workaround of opening an additional tab just to be able to DnD the other one good enough?  I might miss the use case, but I wouldn't think DnDing tabs happens so often it's a deal breaker, nor that it warrants losing that window real estate.

**Comment by cwendling** [comment]
> $ mate-terminal --version
> MATE Terminal 1.18.1

Also note that 1.18 is *very* old, back in 2017.

**Comment by bergstenarn** [comment]
> Yeah, it at least should be an option, I surely wouldn't like the space wasted in my 1-tab windows.
> 
> However, isn't the workaround of opening an additional tab just to be able to DnD the other one good enough? I might miss the use case, but I wouldn't think DnDing tabs happens so often it's a deal breaker, nor that it warrants losing that window real estate.

Making it an option is the right way to go I'd say.

FYI I struggled quite some time trying to organize my terminals in various windows and tabs and several windows contained only one terminal. I tried to find a toggle in the settings about this, but without success. After a while I realized that the workaround is to simply open another tab in windows with only one tab to 'enable' DnD. But my frustration could have been far less with a toggle in the settings to show tabs in windows with only one terminal. Am I the only one who have struggled with this? Maybe I am :-)

**Comment by anishsane** [comment]
+1 for this idea from me as well.  

And it will be useful if it was made as an option (or even editable via CSS if that's easier).
